accord-core/src/main/java/accord/impl/CommandChange.java

```diff
@@ -226,17 +229,17 @@ public Object get(Field field)
         {
             switch (field)
             {
-                case EXECUTE_AT: return executeAt;
```

Comment:
was there a reason for this refactor?  looks like nothing was lost, but also looks arbitrary?

Reply:
To make the field order consistent throughout this class and also broadly consistent with the rest of the codebase (except where otherwise necessary for efficiency, ie participants)